content/productivity/repeat-attack/plugin.js

```diff
@@ -0,0 +1,192 @@
+  ui.centerLocationId(id);
+}
+
+function Attack({ action }) {
+  return html`
+    <div key=${action.id} style=${ActionEntry}>
+      <span>
+        <span
+          style=${{ ...Spacing, ...Clickable }}
+          onClick=${() => centerPlanet(action.payload.srcId)}
+          >${action.payload.srcId.substring(5, 10)}</span
+        >
+        →
+        <span
+          style=${{ ...Spacing, ...Clickable }}
+          onClick=${() => centerPlanet(action.payload.syncId)}
+          >${action.payload.syncId.substring(5, 10)}</span
+        ></span
+      >
+      <button
+        onClick=${() => {
+          op.delete(action.id);
+        }}
+      >
+        ✕
+      </button>
+    </div>
+  `;
+}
+
+function AddAttack() {
+  let [planet, setPlanet] = useState(ui.getSelectedPlanet());
+  let [source, setSource] = useState(false);
+  let [target, setTarget] = useState(false);
+  let onClick = () => {
+    setPlanet(ui.getSelectedPlanet());
+  };
+  useLayoutEffect(() => {
+    window.addEventListener("click", onClick);
+    return () => {
+      window.removeEventListener("click", onClick);
+    };
+  }, [onClick]);
+
+  function createAttack(source, target) {
+    op.pester(source.locationId, target.locationId);
+  }
+
+  return html`
+    <div>
+      <button
+        style=${VerticalSpacing}
+        onClick=${() => {
+          setSource(planet);
+        }}
+      >
+        Set Source
+      </button>
+      <span style=${{ ...Spacing }}
+        >${source ? source.locationId.substring(5, 10) : ""}</span
+      >
+    </div>
+    <div>
+      <button
+        style=${VerticalSpacing}
+        onClick=${() => {
+          setTarget(planet);
+        }}
+      >
+        Set Target
+      </button>
+      <span style=${{ ...Spacing }}
+        >${target ? target.locationId.substring(5, 10) : ""}</span
+      >
+    </div>
+    <button
+      style=${VerticalSpacing}
+      onClick=${() => createAttack(source, target)}
+    >
+      submit
+    </button>
+  `;
+}
+
+function AttackList() {
+  const [actions, setActions] = useState(op.actions);
+
+  let actionList = {
+    maxHeight: "100px",
+    overflowX: "hidden",
+    overflowY: "scroll",
+  };
+
+  let actionsChildren = actions
+    .filter((a) => a.type == "PESTER")
+    .map((action) => {
+      return html`<${Attack} action=${action}></${Attack}>`;
+    });
+
+  return html`
+    <h1 style=${VerticalSpacing}>Create a Recurring Attack</h1>
+
+    <i style=${VerticalSpacing}
+      >When the source planet energy is >75% it will launch an attack
+    </i>
+    <${AddAttack} />
+    <h1 style=${VerticalSpacing}>
+      Recurring Attacks
+      <button
+        style=${{ float: "right" }}
+        onClick=${() => setActions(op.actions)}
+      >
+        refresh
+      </button>
+    </h1>
+    <div style=${{ ...actionList, ...VerticalSpacing }}>
+      ${actionsChildren.length ? actionsChildren : "No Actions."}
+    </div>
+  `;
+}
+
+function App() {
+  return html` <${AttackList} /> `;
```

Comment:
I was having troubles with spacing around element tags in my code. 🤔 
```suggestion
  return html`<${AttackList} />`;
```